spec.html

```diff
@@ -6226,6 +6226,19 @@ <h1>IteratorClose ( _iteratorRecord_, _completion_ )</h1>
       </emu-alg>
     </emu-clause>
 
+    <emu-clause id="sec-ifabruptcloseiterator" aoid="IfAbruptCloseIterator">
+      <h1>IfAbruptCloseIterator ( _value_, _iteratorRecord_ )</h1>
+      <p>IfAbruptCloseIterator is a shorthand for a sequence of algorithm steps that use an Iterator Record. An algorithm step of the form:</p>
+      <emu-alg>
+        1. IfAbruptCloseIterator(_value_, _iteratorRecord_).
+      </emu-alg>
+      <p>means the same thing as:</p>
+      <emu-alg>
+        1. If _value_ is an abrupt completion, return ? IteratorClose(_iteratorRecord_, _value_).
+        1. Else if _value_ is a Completion Record, set _value_ to _value_.[[Value]].
```

Comment:
FYI @bakkot, this should be assumed to be a completion record after completion value reform.

Reply:
Well, this matches all the other `IfAbrupt` shorthands.

Reply:
Yup, I'll fix all of these when I get to this.